src/implementations/vanilla/src/helpers/js/_core.js

```diff
@@ -1,111 +1,127 @@
-var Icons = require('../../basics/icons/icons.js');
-var Mustache = require('mustache');
-require('./polyfills.js');
+var Icons = require("../../basics/icons/icons.js");
```

Comment:
Are there any meaningful changes to `_core.js`? Or is this all just auto-format?

Reply:
nope...just autoformat.